src/js/utils/dom-data.js

```diff
@@ -23,7 +32,7 @@ const elData = {};
  * @constant
  * @private
  */
-const elIdAttr = 'vdata' + (new Date()).getTime();
+const elIdAttr = 'vdata' + Math.floor(now());
```

Comment:
looks like we can't call `now()` detached from it
```suggestion
const elIdAttr = 'vdata' + Math.floor(window.performance && window.performance.now() || Date.now());
```

And then remove line 8-14

Reply:
wow.. 'suggested change' option in github? first time i see this, nice stuff. Will make that change, thanks.

Reply:
yeah, it's pretty nice for single line changes

Reply:
made the change but... i'm not sure if we should change the value in dom-data. Maybe some applications rely on the fact its ms since epoch whereas it now will be 1000 ms or so in general (amount of time since page render). Your call.

Reply:
I can't imagine that anyone would rely on this vdata info and if they are they probably shouldn't.